x-pack/plugins/enterprise_search/public/applications/app_search/components/multi_input_rows/index.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export { CurationQueries } from './curation_queries';
+export { MultiInputRows } from './multi_input_rows';
```

Comment:
Just to clarify:

- This is indeed essentially just the the `CurationQueries` component but relabelled, I basically moved the entire folder out to its own top-level component and did a bunch of find&replacing to more generic names.
- I'm not super in love with the `MultiInputRows` name lol, it's just what came to mind. Definitely open to ideas if y'all have them